app/controllers/follow_requests_controller.rb

```diff
@@ -67,4 +68,10 @@ def set_follow_request
     def follow_request_params
       params.require(:follow_request).permit(:recipient_id, :sender_id, :status)
     end
+
+    def ensure_current_user_is_owner
```

Comment:
Looks like this method is covering both sender and recipient, could also make one for @follow_request.recipient, but doesn't look necessary right now